pages/about.html

```diff
@@ -9,34 +9,38 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
-    <title>Intro to Blockchain</title>
+    <title>Blockchain courses</title>
 </head>
 <body>
-    <header>
+    <header class="bg-container-light">
         <section class="bg-dark d-none d-lg-block mb-0">
             <div class="d-flex align-items-center justify-content-end py-2">
-                
+                <a href="#"><img src="/images/social-icons/Linkedinicon.svg" class="w-75"/></a>
+                <a href="#"><img src="/images/social-icons/twitter.svg" class=" w-25 mx-3"/></a>
                 <a class="nav-link link-secondary">English</a>
                 <a class="nav-link link-secondary mx-3">My page</a>
                 <a class="nav-link link-secondary me-5">Logout</a>
             </div> 
         </section>
         <nav class="navbar navbar-expand-lg mt-0">
             <div class="container">
-              
+              <a class="navbar-brand d-none d-lg-block" href="/index.html"><img src="/images/main-logo.png" class="img-fluid main-logo"/>Blockchain Intro</a>
```

Comment:
The link to the home page is broken. It might be an error due to the relative path you have used here. Remember, your about page is in another directory, so you need to go back one step to refer to the index file.